recipe/meta.yaml

```diff
@@ -20,6 +20,7 @@ build:
 
 requirements:
   build:
+    - gnuconfig  # [unix]
     - {{ compiler('cxx') }}  # [not win]
     - {{ compiler('c') }}    # [not win]
     - vs2017_win-64          # [win64]
```

Comment:
Build is failing now because of the mesa CDTs (below), which AFAIK are unnecessary and shouldn't even work on OSX.